molecule/config-values-test/converge.yml

```
@@ -79,6 +101,19 @@
       - custom_dashboards[0].metadata.name == 'kiali'
       fail_msg: "Must have 1 custom_dashboard: {{ custom_dashboards }}"
 
+  - name: Make sure the custom health config made it to the config map
+      assert:
+        that:
+          - kiali_configmap.health_config.rates | length == 2
+          - kiali_configmap.health_config.rates[0].namespace == "bookinfo"
+          - kiali_configmap.health_config.rates[0].tolerance | length == 2
+          - kiali_configmap.health_config.rates[0].tolerance[1].failure == 10
+          - kiali_configmap.health_config.rates[0].tolerance[1].code == "[45]\\d[^\\D4]"
+          - kiali_configmap.health_config.rates[1].namespace == ".*"
+          - kiali_configmap.health_config.rates[1].tolerance | length == 2
+          - kiali_configmap.health_config.rates[1].tolerance[0].degraded == 30
+          - kiali_configmap.health_config.rates[1].tolerance[1].code == "[5]\\d\\d"
```

lists should be indented at the same level, not doubly-indented, to maintain consistency. This is minor, I'm approving anyway. One day I'll go in and reformat this and the `custom_health_config` definition above. :) 

```suggestion
        that:
        - kiali_configmap.health_config.rates | length == 2
        - kiali_configmap.health_config.rates[0].namespace == "bookinfo"
        - kiali_configmap.health_config.rates[0].tolerance | length == 2
        - kiali_configmap.health_config.rates[0].tolerance[1].failure == 10
        - kiali_configmap.health_config.rates[0].tolerance[1].code == "[45]\\d[^\\D4]"
        - kiali_configmap.health_config.rates[1].namespace == ".*"
        - kiali_configmap.health_config.rates[1].tolerance | length == 2
        - kiali_configmap.health_config.rates[1].tolerance[0].degraded == 30
        - kiali_configmap.health_config.rates[1].tolerance[1].code == "[5]\\d\\d"
```

---

Fixed ;)

---

> and the `custom_health_config` definition above

but not this :) (i.e. lines 55-69)